x-pack/plugins/observability/public/routes/index.tsx

```diff
@@ -112,4 +112,17 @@ export const routes = {
       }),
     },
   },
+  '/exploratory-view/multi': {
```

Comment:
I think we should comment this out completely, so it is completely inaccessible by our users until it is ready.